### ISSUE TITLE
Module not found - NextJs 13

### ISSUE THREAD

**Issue by andrefelipeschulle** [lucide-icons/lucide#1532]
hello everything is fine ? I am having problems with some specific icons in the NextJs 13 AppRouter, for example, Grid2x2 or Grid3x3 icons are showing the Module not found error, but other icons like ChevronUp work correctly, I found something in vercel issues, but I couldn't make it work here.

<img width="1268" alt="Captura de Tela 2023-08-25 às 00 15 25" src="https://github.com/lucide-icons/lucide/assets/33896558/e624dd73-c029-4f87-b129-fbfacac57e6b">

the lucide-react/dist/esm/icons/grid-2-x-2.js file does not exist in the folder, it has another name, grid2x2.js

## Prerequisites

* Version: 0.268.0
* Are you running from source/main: yes
* Are you using a released build: no
* Operating system:  Mac OSx - M1
* Bits: 64 Bits

## Step to reproduce

```
"use client";

import * as Collapsible from "@radix-ui/react-collapsible";
import * as ToggleGroup from "@radix-ui/react-toggle-group";
import { useState } from "react";
import { ChevronDown, ChevronUp, Columns, Grid2x2 } from "lucide-react";
import Link from "next/link";

importer correctly -> ChevronDown, ChevronUp, Columns

Module not found: Can't resolve -> Grid2x2
```

**Comment by andrefelipeschulle** [comment]
importing this way works but the error of the types of the typescript

```jsx
import Grid2x2 from "lucide-react/dist/esm/icons/grid-2x2";
```

**Comment by karsa-mistmere** [comment]
Possible duplicate of #1482

**Comment by andrefelipeschulle** [comment]
> Possible duplicate of #1482

I had already seen this issue but I didn't find anything that would help me solve this problem, I believe it is something specific to the NEXT configs.

**Comment by karsa-mistmere** [comment]
I'm not very familiar with Next.js, but maybe you should provide your next.config.js then, so that others will be able to help with it.

**Comment by andrefelipeschulle** [comment]
aí está, não há nada diferente na configuração Next, é padrão.

```javascript
/** @type {import('next').NextConfig} */
const nextConfig = {};

module.exports = nextConfig;

```

**Comment by karsa-mistmere** [comment]
Sorry, I must have misunderstood your statement about there being something specific about the Next.js config.
Which version of Next.js are you on?

**Comment by andrefelipeschulle** [comment]
sorry, I must have expressed myself badly, I believe something in the next project configuration, 

https://github.com/vercel/next.js/pull/53697

must be causing this error in the import, in the link above, we see some modifications that were made in the Next project but that are causing this possible Bug.

**Comment by ericfennis** [comment]
Look like to weird case transformation by Moduliaze imports in Nextjs. 
Created a fix for these edge cases in #1549.
